pkg/volume/csi/csi_plugin.go

```diff
@@ -124,7 +124,7 @@ func (h *RegistrationHandler) RegisterPlugin(pluginName string, endpoint string,
 	})
 
 	// Get node info from the driver.
-	csi, err := newCsiDriverClient(csiDriverName(pluginName))
+	csi, err := newCsiDriverClient(csiDriverName(pluginName), nil)
```

Comment:
Why is there no TracerProvider here?

Reply:
I think since the function `RegisterPlugin` is  be used by kubelet's pluginmanager, which also managed the deviceplugin and draplugin. So if the function signature is changed (add traceprovider), the deviceplugin and draplugin may also should be changed.

Reply:
please use a no-op here, then.